src/System.Diagnostics.Debug/src/System/Diagnostics/DebuggerVisualizerAttribute.cs

```diff
@@ -0,0 +1,98 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+namespace System.Diagnostics
+{
+
+    /// <summary>
+    /// Signifies that the attributed type has a visualizer which is pointed
+    /// to by the parameter type name strings.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Struct | AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = true)]
+    [System.Runtime.InteropServices.ComVisible(true)]
```

Comment:
`ComVisible` can be removed. See https://github.com/dotnet/corefx/pull/12113#discussion_r82315784